src/info.json

```diff
@@ -2,19 +2,24 @@
     "name": "subspace_storage",
     "variants": [
         {
-            "version": "1.99.18",
+            "version": "2.0.17",
```

Comment:
Should these versions not be `2.17.0` which then allows us to release patches as `2.17.2` ?
(This also applies to clusterio_lib, although I did not notice while doing that review)

Reply:
I have been going a bit back on forth on it. I think its more useful to have minor versions. As I set it now, the patch versions (releases for different factorio versions) are feature wise identical, which is correct semver. It also makes the sorting make more sense, putting the newest release at the top and not the newest factorio version.

Reply:
Oh I see, yeah that does make more sense now that you have explained it.